examples/siro_sandbox/README.md

```diff
@@ -43,7 +43,7 @@ habitat.environment.iterator_options.cycle=False \
 habitat_baselines.evaluate=True \
 habitat_baselines.num_environments=1 \
 habitat_baselines.eval.should_load_ckpt=False \
-habitat_baselines.rl.agent.num_pool_agents_per_type='[1,1]' \
+habitat_baselines.rl.agent.num_pool_agents_per_type=[1,1] \
```

Comment:
In installation, we should clarify if we require both `habitat-lab` and `habitat-baselines` packages. You might also try to explain how it's not strictly necessary for the tool running at `examples/hitl_tool/hitl_tool.py` to import from the version of habitat-lab/baselines that lives at the parent directory (it just needs to be importable in your current environment), although I'm not sure how to cleanly explain this.